.github/workflows/validate_charts.yml

```diff
@@ -0,0 +1,30 @@
+name: validate charts
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  validate-helm-charts:
+    runs-on: ubuntu-20.04
+    container: hypertrace/helm-gcs-packager:0.3.0
```

Comment:
Should we be providing pull credentials for dockerhub images to avoid rate limiting, or does GH have some work around for that?

Reply:
yeah